src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -84,6 +91,21 @@ public MainWindow(Stage primaryStage, Config config, UserPrefs prefs, Logic logi
         registerAsAnEventHandler(this);
     }
 
+    /**
+     * Changes the stylesheet used by GUI.
+     */
+    public void changeTheme(int theme) {
+        String brightTheme = "view/BrightTheme.css";
+        String darkTheme = "view/DarkTheme.css";
+        if (theme == 0) {
```

Comment:
Something to ponder: is this code extensible? What happens if down the line, you want to add new themes? What would be a better way to implement this?